lib/action_view/component/monkey_patch.rb

```diff
@@ -25,3 +31,11 @@ def render(options = {}, args = {}, &block)
 
   prepend RenderMonkeyPatch
 end
+
+class ActiveRecord::Base
```

Comment:
I checked how active record handles this, but it seems like it does not. It's a responsibility of ActiveModel, so do you think we should use `ActiveModel::Model` here instead?

What would be even better I think is to update this module https://github.com/rails/rails/blob/master/activemodel/lib/active_model/conversion.rb with this new `to_component_class` method.

Reply:
@tomasc it seems like this would solve your concern right?